### PR TITLE
Improve README.md formatting

### PR DESCRIPTION
**Pull request by Bmkiel**
Greetings, 

Looking over the `README.md` file I noticed a few areas for minor improvement such as: enclosing variables in backticks, minor punctuation fixes, and formatting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-While this plugin is part of the Foreman project, it can only be used with Katello as Foreman is not content aware without the Katello plugin.
+Foreman project plugin for Pulp allowing Katello hosts to interact with Pulp application services for content management. While this plugin is part of the Foreman project, it can only be used with Katello as Foreman is not content aware without the Katello plugin.
 
 ## Getting Started
 
 The Foreman project provides documentation on this specific plugin installation from multiple methods, distribution specific RPM, distribution specific DEB and direct manual build from source.
-The plugin can also be installed as part of the foreman-installer puppet class with the "--[no-]enable-foreman-proxy-plugin-pulp" option.
-Documentation can be found here for how to install a stand alone smart proxy https://theforeman.org/manuals/latest/index.html#4.3.1SmartProxyInstallation
+
+The plugin can also be installed as part of the foreman-installer puppet class with the `--[no-]enable-foreman-proxy-plugin-pulp` option.
```
Comment (ekohl) [inline README.md:9]: This is not true anymore, we've removed this and now wrap it in a different class. I'd leave it out.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
 # Foreman Pulp Plugin
 
-Foreman project plugin for Pulp allowing Katello hosts to interact with Pulp application services for content management
-While this plugin is part of the Foreman project, it can only be used with Katello as Foreman is not content aware without the Katello plugin.
+Foreman project plugin for Pulp allowing Katello hosts to interact with Pulp application services for content management. While this plugin is part of the Foreman project, it can only be used with Katello as Foreman is not content aware without the Katello plugin.
 
 ## Getting Started
 
 The Foreman project provides documentation on this specific plugin installation from multiple methods, distribution specific RPM, distribution specific DEB and direct manual build from source.
```
Comment (ekohl) [inline README.md:7]: In this particular case we don't build a DEB. I think that could have been copy pasted from a template.

**Review by ekohl — COMMENTED** [review]
Thanks for this. Would you mind fixing the additional cases? I'd also be happy to merge your change since it's an improvement already.

**Comment by Bmkiel** [comment]
Hey @ekohl, thanks for the suggestions. I updated with the additional changes. Let me know what you think.

**Review by ekohl — APPROVED** [review]
Thanks!

**Pull request merged**